src/main/java/uk/gov/justice/digital/delius/data/api/StaffHuman.java

```diff
@@ -21,11 +21,6 @@
     private String forenames;
     @ApiModelProperty(value = "Family name", example = "Hancock")
     private String surname;
-    @ApiModelProperty(value = "Staff Grade", example = "PO")
-    private String staffGrade;
-    @ApiModelProperty(value = "staff identifier", example = "123456")
-    private Long staffIdentifier;
-
```

Comment:
Should we be removing these as it might affect other endpoints?

Reply:
I added these in a previous commit for this PR

Reply:
Ah my bad.  I was looking at the commit changes rather than the PR changes.